lib/ocpp/v201/connectivity_manager.cpp

```diff
@@ -133,6 +130,9 @@ void ConnectivityManager::init_websocket() {
         EVLOG_AND_THROW(std::runtime_error("ChargePointId must not contain \':\'"));
     }
 
+    // cache the network profiles on initialization
+    cache_network_connection_profiles();
+
     const auto network_connection_priorities = ocpp::get_vector_from_csv(
```

Comment:
The network configuration priority should also be part of this caching I assume?